libs/vscode/nx-cloud-view/src/lib/nx-cloud-service/nx-cloud-service.ts

```diff
@@ -233,16 +233,14 @@ export class NxCloudService extends StateBaseService<InternalState> {
           try {
             if (lt(coerce(nxVersion.full) ?? '', '12.0.0')) {
               promisify(exec)(
-                `${getPackageManagerCommand().addDev} @nrwl/nx-cloud@${
+                `${getPackageManagerCommand().addDev} @nx/nx-cloud@${
```

Comment:
this is not going to work because there's not `@nx/nx-cloud` with v12 right?

Reply:
you're right. The migration that changed @nrwl to @nx was a bit overzealous, and I'm going to try the migration again with the latest beta today.